Apps/AdminWebClient/src/ClientApp/src/services/interfaces.ts

```diff
@@ -69,6 +70,11 @@ export interface ISupportService {
     ): Promise<MessageVerification[]>;
 }
 
+export interface ICovidSupportService {
+    initialize(http: IHttpDelegate): void;
+    searchByPHN(phn: string): Promise<CovidCardRequestResult>;
```

Comment:
On the controller side there are three paths:
Retrieve covid info -> Patient Info + Immz Events
Retrieve Doc -> base 64 doc + file name (similar to webclient)
Mail Doc -> boolean (true if sucessful)